src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-control-plane-monitoring.mdx

```diff
@@ -52,14 +52,14 @@ By default, our [Helm Chart](/docs/kubernetes-pixie/kubernetes-integration/insta
 
 #### `hostNetwork` and `privileged` [#hostnetwork-privileged]
 
-Most users and Kubernetes distributions configure the control plane metrics endpoints to listen only in the loopback interface, i.e. `localhost`. For this reason, the control plane component is deployed with `hostNetwork: true` by default when `privileged` is set to `true` (the default).
+In versions lower than v3, when the integration is deployed using `privileged: false`, the `hostNetwork` setting for the control plane component will be also be set to `false`.
```

Comment:
```suggestion
In versions older than v3, when the integration is deployed using `privileged: false`, the `hostNetwork` setting for the control plane component will be also be set to `false`.
```

Reply:
Solved in [#c326625](https://github.com/newrelic/docs-website/pull/7322/commits/c32662541d45e88c090fd3eeecea3935bdd6e660)